src/AutoPilotPlugins/PX4/CameraComponentSummary.qml

```diff
@@ -14,7 +14,7 @@ FactPanel {
     QGCPalette { id: qgcPal; colorGroupEnabled: enabled }
     FactPanelController { id: controller; factPanel: panel }
 
-    property Fact _camTriggerMode:  controller.getParameterFact(-1, "TRIG_MODE")
+    property Fact _camTriggerMode:  controller.getParameterFact(-1, "TRIG_MODE", false)
```

Comment:
@LorenzMeier 
If there is a case where TRIG_MODE is not available at all, the entire Camera Setting panel should be hidden. If that's the case this code should be changed:
https://github.com/mavlink/qgroundcontrol/blob/master/src/AutoPilotPlugins/PX4/PX4AutoPilotPlugin.cc#L129-L131
And use the same logic as the one used right below it for the WiFi Module. Let me know if this is the case and I can change it.
I added the last boolean to `controller.getParameterFact()` to deal with the rest of the parameters that only exist if TRIG_MODE is disabled. If the entire functionality is missing, we might as well don't show the panel at all.